graphql/resolvers/directives.js

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import path from 'path';
+import { forEachField } from 'graphql-tools';
+import { getArgumentValues } from 'graphql/execution/values';
+import jwt from 'jsonwebtoken';
+import debug from 'debug';
+
+import { AuthorizationError } from '../errors';
+
+const log = debug('fcc:resolvers:directives');
+const certPath = process.cwd() + '/public.pem';
+const cert = fs.readFileSync(certPath);
+
+function isAuth(ctx) {
```

Comment:
Should we just call this `validateWebtoken` or something else that is more descriptive of what we are actually doing here?

Reply:
Of course.